src/ClusterBootstrap/services/job-insighter/job-insighter-cronjob.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: batch/v1beta1
+kind: CronJob

```

Comment:
Why job instead of long running service?

Reply:
Will change to a service